pallets/parachain-staking/src/lib.rs

```diff
@@ -507,6 +538,12 @@ pub mod pallet {
 	pub(crate) type CandidateInfo<T: Config> =
 		StorageMap<_, Twox64Concat, T::AccountId, CandidateMetadata<BalanceOf<T>>, OptionQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn candidate_rounds_info)]
+	/// Stores the last round in which a collator produced blocks
+	pub(crate) type CandidateRoundsInfo<T: Config> =
+		StorageMap<_, Twox64Concat, T::AccountId, CandidateLastRound<RoundIndex>, OptionQuery>;
```

Comment:
Can we use the the `RoundIndex` directly here? Something like `CollatorLastActive<T::AccountId, RoundIdex>`